File: src/test/java/com/baulsupp/oksocial/services/twitter/TwitterAuthInterceptorTest.java
package com.baulsupp.oksocial.services.twitter;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertTrue;

public class TwitterAuthInterceptorTest {
  TwitterAuthInterceptor auth = new TwitterAuthInterceptor();

  @Test public void testHosts() {
    assertTrue(auth.hosts().contains("api.twitter.com"));
  }
}

File: src/test/kotlin/com/baulsupp/oksocial/services/twitter/TwitterAuthInterceptorTest.kt
package com.baulsupp.oksocial.services.twitter

import org.junit.Test
import kotlin.test.assertTrue


class TwitterAuthInterceptorTest {
    internal var auth = TwitterAuthInterceptor()

    @Test
    fun testHosts() {
        assertTrue(auth.hosts().contains("api.twitter.com"))
    }
}
